packages/babel-plugin/src/utils/object-property-to-string.ts

```diff
@@ -56,6 +56,12 @@ const expressionToString: ExpressionToString = (expression, meta) => {
   // handles {[key]: 'value'}
   if (t.isIdentifier(expression)) {
     const evaluatedExpression = evaluateExpression(expression, meta);
+    if (evaluatedExpression.value === expression) {
+      throw new Error(
+        `Cannot statically evaluate the value of "${expression.name}"; ` +
+          `are you importing this variable from an external package?`
```

Comment:
Do you mean `@compiled/babel-plugin` can’t static evaluate a variable imported from an external package?

Reply:
Ah you're right, thank you for mentioning this - I'll remove the second part of the error until we find what causes this recursion